frigate/object_processing.py

```diff
@@ -165,6 +166,10 @@ def update(self, current_frame_time, obj_data):
         if self.computed_score > self.top_score:
             self.top_score = self.computed_score
         self.false_positive = self._is_false_positive()
+        if not self.active == self.is_active(): 
```

Comment:
this is incorrect, it can only be a significant change if the object is not a false positive, this condition should stay where it was before

Reply:
Lost my comment - I stored this in a variable and moved the check back into the code block protected by the false_positive check.